packages/core/src/utils/oca.ts

```diff
@@ -11,32 +11,73 @@ import {
 import { W3cCredentialDisplay } from '../modules/openid/types'
 import { BrandingOverlay } from '@bifold/oca'
 
-export const buildFieldsFromAnonCredsCredential = (credential: CredentialExchangeRecord): Array<Field> => {
-  return credential?.credentialAttributes?.map((attr) => new Attribute(attr)) || []
+export type FiledExt = {
```

Comment:
Just a question, was this meant to be FieldExt rather than FiledExt? I'm not super familiar with the OpenID stuff